src/script/main/app.ts

```diff
@@ -31,7 +31,7 @@ import {Runtime} from '@wireapp/commons';
 
 import {getLogger, Logger} from 'Util/Logger';
 import {t} from 'Util/LocalizerUtil';
-import {arrayToBase64, checkIndexedDb, createRandomUuid} from 'Util/util';
+import {arrayToBase64, checkIndexedDb, createRandomUuid, isMLSDisabledOnElectron} from 'Util/util';
```

Comment:
as much as possible, we try not to mention electron in the webapp. It doesn't need to be aware of it. 
Let's call it `canSupportMLS`

Reply:
sure 👍